win32ss/gdi/dib/dib32bpp.c

```diff
@@ -365,8 +366,7 @@ DIB_32BPP_BitBltSrcCopy(PBLTINFO BltInfo)
       {
         /* SourceBits points to bottom-left pixel for lDelta < 0 and top-left for lDelta > 0 */
         SourceBits = (PBYTE)BltInfo->SourceSurface->pvScan0
-          + ((BltInfo->SourcePoint.y
-          + DestHeight - 1) * BltInfo->SourceSurface->lDelta)
+          + ((BltInfo->SourcePoint.y + (LONG)DestHeight - 1) * BltInfo->SourceSurface->lDelta)
```

Comment:
`DestHeight` is already a LONG in your PR. This `(LONG)` typecast is meaningless.